csrc/runtime/compiled_kernel.cpp

```diff
@@ -1198,6 +1199,8 @@ NVF_API CompiledKernel::CompiledKernel(
           {},
           {}) {}
 
+NVF_API CompiledKernel::~CompiledKernel() = default;
```

Comment:
Because of `std::unique_ptr<GpuLower> CompiledKernel::lowered_`, to avoid including `lower2device.h` in compiled_kernel.h, this destructor needs to be defined here.